Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/CoreRESTAPI.js

```diff
@@ -24,16 +51,18 @@ sendMultipart = function (uri, entryID, body) {
         if (auth_id == ''){
             throw 'Auth ID must be provided.';
     }
+    var headers = {}
+    if (params.auth_method == 'standard'){
```

Comment:
what will happen if auth_method is empty?
the integration was installed before with standard api key, and now the pack was updated to the latest version

Reply:
right, fixed.